nestkernel/connection_manager.cpp

```diff
@@ -1163,13 +1157,7 @@ nest::ConnectionManager::get_connections(
 #ifdef _OPENMP
 #pragma omp critical( get_connections )
 #endif
-        {
-          while ( !conns_in_thread.empty() )
-          {
-            connectome.push_back( conns_in_thread.front() );
-            conns_in_thread.pop_front();
-          }
-        }
+        append_move_connections_( conns_in_thread, connectome );
```

Comment:
@heplesser Thanks for the comments!
Maybe you also like something like this. Looks good from my point of view. :)
```
static inline std::deque< nest::ConnectionID >&
operator<<( std::deque< nest::ConnectionID >& out,
  std::deque< nest::ConnectionID >& in )
{
  while ( not in.empty() )
  {
    out.push_back( in.front() );
    in.pop_front();
  }

  return out;
}
```

And then just write
`connectome << conns_in_thread;`